harmonic/ros2_spawn_model.md

```diff
@@ -0,0 +1,61 @@
+# Spawn a Gazebo model from ROS 2
+
+Gazebo will spawn all the models included in the provided world file at startup.
+Additionally, it's possible to spawn new models at any time. To do so using ROS
+we have provided the following mechanisms:
+
+## Spawn a model using the launch file included in `ros_gz_sim`.
+
+The package `ros_gz_sim` contains a launch file named
+`ros_gz_spawn_model.launch.py`. You can use it to spawn a new model into an
+existing simulation. Here's an example:
+
+```bash
+ros2 launch ros_gz_sim gz_spawn_model.launch.py world:=empty file:=/home/caguero/ros_gz_ws/src/ros_gz/ros_gz_sim_demos/models/vehicle/model.sdf name:=my_vehicle x:=5.0 y:=5.0 z:=0.5
```

Comment:
This can be made more generic by using `ros2 pkg`.
```suggestion
ros2 launch ros_gz_sim gz_spawn_model.launch.py world:=empty file:=$(ros2 pkg prefix --share ros_gz_sim_demos)/models/vehicle/model.sdf name:=my_vehicle x:=5.0 y:=5.0 z:=0.5
```